GameServerCore/Domain/GameObjects/IChampion.cs

```diff
@@ -17,6 +17,7 @@ public interface IChampion : IObjAiBase
         void TeleportTo(float x, float y);
         void UpdateSkin(int skinNo);
         int GetChampionHash();
+        // Is this needed? We also have StopMovement() in ObjAiBase
         void StopChampionMovement();
```

Comment:
I doubt that is needed, but i don't think it should be done in this PR. You might want to open an issue for that.